src/Cache/Cache.php

```diff
@@ -17,6 +22,18 @@ public function __construct(Adapter $adapter)
         $this->adapter = $adapter;
     }
 
+    /**
+     * Toggle case sensitivity of keys inside cache
+     *
+     * @param string $key
+     * @param boolean $value if true, cache keys will be case sensitive
+     * @return mixed
+     */
+    public function setCaseSensitivity(bool $value)
```

Comment:
```suggestion
    public static function setCaseSensitivity(bool $value)
```
I think it makes more sense of this to be a static function